newIDE/app/src/LayersList/LightingLayerDialog.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Dialog from '../UI/Dialog';
+import ColorField from '../UI/ColorField';
+import { Column, Line, Spacer } from '../UI/Grid';
+import InlineCheckbox from '../UI/InlineCheckbox';
+import Text from '../UI/Text';
+import { type RGBColor, type ColorResult } from '../UI/ColorField/ColorPicker';
+import DismissableAlertMessage from '../UI/DismissableAlertMessage';
+
+type Props = {
+  layer: gdLayer,
+  onClose: () => void,
+};
+
+const LightingLayerDialog = (props: Props) => {
+  const [color, setColor] = React.useState<RGBColor>({
+    r: props.layer.getAmbientLightColorRed(),
+    g: props.layer.getAmbientLightColorGreen(),
+    b: props.layer.getAmbientLightColorBlue(),
+  });
+  const [followBaseLayer, setFollowBaseLayer] = React.useState(
+    props.layer.isFollowingBaseLayerCamera()
+  );
+
+  const actions = [
+    <FlatButton
+      label={<Trans>Cancel</Trans>}
+      onClick={props.onClose}
+      key={'Cancel'}
+    />,
+    <FlatButton
+      label={<Trans>Apply</Trans>}
+      primary
+      keyboardFocused
+      onClick={() => {
+        props.layer.setAmbientLightColor(color.r, color.g, color.b);
+        props.layer.setFollowBaseLayerCamera(followBaseLayer);
+        props.onClose();
+      }}
+      key={'Apply'}
+    />,
+  ];
+  return (
+    <Dialog
+      noMargin
+      open={!!props.layer}
+      title={<Trans>Lighting layer settings</Trans>}
+      actions={actions}
+    >
+      <Column>
+        <Line>
```

Comment:
Use `<ColumnStackLayout>` that is a Column that introduces `Spacer` between each item - you then probably don't need Line and Spacer at all.